GetFitLA_v3/app/src/main/java/practicum/getfitla_v3/Exercise.java

```diff
@@ -0,0 +1,65 @@
+package practicum.getfitla_v3;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Work on 3/13/18.
+ */
+
+public class Exercise extends AppCompatActivity {
+
+    RecyclerView recyclerView;
+    //creating instances of adaptor to link to the recyclerview
+    List<detail> exerciseList;
+
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.exercise);
+
+        recyclerView = (RecyclerView) findViewById(R.id.reyclerView);
+        recyclerView.setHasFixedSize(true); //sets a fixed size for the recycler view size, not the elements in the recycler view
+        recyclerView.setLayoutManager(new LinearLayoutManager(this)); //setting the orientation of the recyclerview (by default it is vertical
+
+        exerciseList = new ArrayList<>(); //initializing the array list
+
+        exerciseList.add(
```

Comment:
Ok so as I understand it, this is the hard-coded information that will later be replaced with information pulled directly from Google Sheets. When you do implement the API, these lines will be replaced with calls from a class that you should create to organize the information from the google sheet.